### PR TITLE
completing the ReverseInteger7 task

### PR DESCRIPTION
**Pull request by CHAriina**

### CODE REVIEW

File: ScnSoft-Education/LeetCodeTraining.Tests/ReverseInteger.Tests.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace LeetCodeTraining.Tests
+{
+    public class SolutionTests
+    {
+        [Theory]
+        [InlineData(123, 321)]
+        [InlineData(-123, -321)]
+        [InlineData(120, 21)]
+        [InlineData(1534236469, 0)] // Пример переполнения
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/ReverseInteger.Tests.cs]: use English language

File: ScnSoft-Education/LeetCodeTraining/ReverseInteger7/ReverseInteger.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿namespace LeetCodeTraining.ReverseInteger7;
+public class Solution
+{
+    public int Reverse(int inputNumber)
+    {
+        int reversed = 0;
+
+        while (inputNumber != 0)
+        {
+            int digit = inputNumber % 10;
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/ReverseInteger7/ReverseInteger.cs]: 'digit' and 'inputNumber' are using in 18 line. Move this code in that place

File: ScnSoft-Education/LeetCodeTraining.Tests/ReverseInteger.Tests.cs (hunk 1 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining.Tests/ReverseInteger.Tests.cs]: The Russian language has been replaced

File: ScnSoft-Education/LeetCodeTraining/ReverseInteger7/ReverseInteger.cs (hunk 2 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining/ReverseInteger7/ReverseInteger.cs]: the location of the code lines has been changed

File: ScnSoft-Education/LeetCodeTraining/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "profiles": {
+    "LeetCodeTraining": {
+      "commandName": "Project"
+    },
+    "Профиль 1": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\Arisha\\Desktop\\EDUCATION\\ScnSoft-Education\\LeetCodeTraining\\TwoSum1"
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/Properties/launchSettings.json]: remove this profile file

**Review by dsubotkovsky — CHANGES_REQUESTED** [review]

File: ScnSoft-Education/LeetCodeTraining/Properties/launchSettings.json (hunk 3 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining/Properties/launchSettings.json]: removed

**Pull request merged**